components/routes.jsx

```diff
@@ -2,27 +2,30 @@
 import React from 'react';
 /*eslint-disable no-unused-vars*/
 
-import { Route } from 'react-router';
+import { Route, IndexRoute } from 'react-router';
 import pages from '../data/pages.js';
 import locales from '../public/locales.json';
 
-var routes = (
-  <Route>
-    <Route path="/"/>
-    {
-      Object.keys(locales).map(function(locale) {
-        return Object.keys(pages).map(function(key) {
-          var routeOBJ = {
-            key: key,
-            path: `${locale}${key}`,
-            component: pages[key]
-          };
-          return (
-            <Route {...routeOBJ}/>
-          );
-        });
-      })
+function buildPagesFromData(data) {
+  return Object.keys(data).map(function(pageKey) {
+    if (pageKey === "/") {
+      return (<IndexRoute key={pageKey} component={data[pageKey]}/>);
     }
+    return (<Route key={pageKey} path={pageKey.slice(1, -1)} component={data[pageKey]}/>);
```

Comment:
Split is removing the opening and closing slashes on the urls, just for react router.